windowsBase/windowsBaseAMI.json

```diff
@@ -13,21 +13,22 @@
       "access_key": "{{ user `aws_access_key` }}",
       "secret_key": "{{ user `aws_secret_key` }}",
       "region": "{{ user `REGION` }}",
-      "instance_type": "t2.micro",
-      "source_ami_filter": {
-        "filters": {
-          "virtualization-type": "hvm",
-          "name": "*Windows_Server-2016-English-Full-Base*",
-          "root-device-type": "ebs"
-        },
-        "most_recent": true,
-        "owners": "amazon"
-      },
+      "instance_type": "c4.large",
+      "source_ami": "{{user `SOURCE_AMI`}}",
       "ami_name": "windowsbase-ami-{{user `RES_IMG_VER_NAME_DASH`}}-{{isotime \"2006-01-02-1504\"}}",
       "user_data_file": "./bootstrap_win.txt",
       "communicator": "winrm",
       "winrm_username": "{{ user `WINRM_USERNAME` }}",
-      "winrm_password": "{{ user `WINRM_PASSWORD` }}"
+      "winrm_password": "{{ user `WINRM_PASSWORD` }}",
+      "launch_block_device_mappings": [
+        {
+          "device_name": "/dev/sda1",
+          "volume_type": "io1",
+          "volume_size": "120",
+          "iops": "6000",
```

Comment:
Do we really need provisioned IOPS on this? We are hardly going to be pulling any images here.